pkgs/tools/security/fpm2/default.nix

```diff
@@ -1,20 +1,20 @@
-{ lib, stdenv, fetchurl, pkg-config, gnupg, gtk2
-, libxml2, intltool
+{ lib, stdenv, fetchurl, pkg-config, gnupg, gtk3
+, libxml2, intltool, nettle
 }:
 
 with lib;
```

Comment:
Would be nice if we could remove this

Reply:
WDYT of https://github.com/NixOS/nixpkgs/pull/177287/commits/d82e1663c7e4d3beea77d3d98ba8defa2f937caa ?